### DISCUSSION TITLE
Is there a way to reset a zoom level to default?

### DISCUSSION THREAD

**Discussion by hyperrecursive** [doublecmd/doublecmd#910] [General]
Sorry, my OCD is killing me, I am looking for something like Ctrl+0 in Chrome. :)

**Comment by iglezz** [comment]
Zoom level reset is not possible for now.

**Comment by alexx2000** [comment]
Do you mean zoom level in the image viewer?

**Reply by hyperrecursive** [reply]
Nope, the main file view - pressing Ctrl scrolling with the mouse wheel increases/decreases the font size. I'd like to know if there is a way to reset that font size to default.

**Reply by alexx2000** [reply]
No, because main file view does not have a zoom function. Ctrl + mouse wheel directly changes a font size, same way as [options dialog](https://doublecmd.github.io/doc/en/configuration.html#ConfigFonts).
